packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -121,28 +121,28 @@ module.exports = {
         },
         {
             name: 'output',
-            usage: '--output <path to output directory> e.g. ./dist/',
+            usage: '--output <path to output directory>',
             alias: 'o',
             group: OUTPUT_GROUP,
             type: String,
-            description: 'Output location of the file generated by webpack',
+            description: 'Output location of the file generated by webpack e.g. ./dist/',
             link: 'https://webpack.js.org/concepts/#output',
         },
         {
             name: 'plugin',
-            usage: '--plugin',
+            usage: '--plugin <pluginName>',
             group: ADVANCED_GROUP,
             type: String,
-            description: 'Load a given plugin',
+            description: 'Load a given plugin e.g. HtmlWebpackPlugin',
```

Comment:
This is only to generate plugin iirc?

Reply:
I assume you're speaking about the `plugin` command which is used to scaffold a plugin.
https://github.com/webpack/webpack-cli/blob/49620e0070a1be9287b493fe22bfcfab650341f3/packages/webpack-cli/lib/utils/cli-flags.js#L42

Reply:
It isn't the case here.

Reply:
Ah okay yes